RRS/customer/urls.py

```diff
@@ -10,8 +10,8 @@
         name  = 'index'
     ),
     url(
-        regex = r'^(\d+)/$',
+        regex = r'restaurant/(\d+)$',
         view = views.restaurant,
-        name="restaurant"
+        name="view-restaurant"
```

Comment:
Consider the use:
 - Namespaced: `{% url customer:restaurant 1 %}` 
 - or `{% url customer:view-restaurant 1 %}`
 - Without namespace: `{% url customer.views.restaurant 1 %}`

I think the first example is readable enough. The second is more readable but might confuse the developer that doesn't check urls.py first. If deciding names becomes an issue, we can choose not to use namespaces:))